libs/@local/hash-isomorphic-utils/src/temporal-types.ts

```diff
@@ -3,7 +3,8 @@ import { Subtype } from "@local/advanced-types/subtype";
 import { Entity } from "@local/hash-graph-client";
 import {
   AccountId,
-  EntityPropertiesObject,
+  BaseUrl,
+  EntityPropertyValue,
   OwnedById,
 } from "@local/hash-subgraph";
```

Comment:
I think it's because I thought I'd have to cast it at some point anyway, but I've remembered I can use `mapGraphApiSubgraphToSubgraph` to convert it instead. I'll do this in the follow-up branch because it'll probably update quite a few places scattered around the plugin